openid_connect.js

```diff
@@ -7,7 +7,38 @@ var newSession = false; // Used by oidcAuth() and validateIdToken()
 
 export default {auth, codeExchange, validateIdToken, logout};
 
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function isVarDefined(variable) {
```

Comment:
The name does not quite match the content. Not even sure if this function is really needed.

Reply:
agree, moved this logic to "waitForSessionSync" func.